package.json

```diff
@@ -15,17 +15,27 @@
     "url": "https://github.com/microsoft/BotFramework-WebChat/issues"
   },
   "homepage": "https://github.com/microsoft/BotFramework-WebChat#readme",
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged --no-stash --config lint-staged.config.js"
-    }
-  },
   "keywords": [],
   "scripts": {
     "bootstrap": "lerna bootstrap --ci",
     "build": "lerna run --ignore playground --stream build",
     "coveralls": "cat ./coverage/lcov.info | coveralls",
-    "eslint": "lerna run --parallel --stream eslint -- ",
+    "eslint": "echo TEMPORARILY DISABLE: npm run precommit",
```

Comment:
I feel like the scripts section is getting long enough that we should extract it out into a scripts file